cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml

```diff
@@ -369,7 +372,8 @@ data:
       matches [{ "_SYSTEMD_UNIT": "kubelet.service" }]
       <storage>
         @type local
-        persistent true
+        persistent false
```

Comment:
why do you remove persistence?
its there to be able to pickup the buffer after a container crash.
otherwise you might loose some logs.